examples/USBH_MIDI/USBH_MIDI_dump/USBH_MIDI_dump.ino

```diff
@@ -60,15 +54,14 @@ void MIDI_poll()
   uint8_t bufMidi[64];
   uint16_t  rcvd;
 
-  if (Midi.vid != vid || Midi.pid != pid) {
-    sprintf(buf, "VID:%04X, PID:%04X", Midi.vid, Midi.pid);
+  if (Midi.idVendor() != vid || Midi.idProduct() != pid) {
+    vid = Midi.idVendor();
+    pid = Midi.idProduct();
+    sprintf(buf, "VID:%04X, PID:%04X", vid, pid);
     Serial.println(buf);
-    vid = Midi.vid;
-    pid = Midi.pid;
   }
   if (Midi.RecvData( &rcvd,  bufMidi) == 0 ) {
-    uint32_t time = (uint32_t)millis();
-    sprintf(buf, "%04X%04X: ", (uint16_t)(time >> 16), (uint16_t)(time & 0xFFFF)); // Split variable to prevent warnings on the ESP8266 platform
```

Comment:
This change causes the build to fail on the ESP platforms: https://travis-ci.org/felis/USB_Host_Shield_2.0/jobs/357797072#L1470.